Src/library/cURL/cURL.ecf

```diff
@@ -23,38 +23,60 @@
 				<platform excluded_value="windows"/>
 			</condition>
 		</external_include>
-		<external_object location="$(ISE_LIBRARY)/library/cURL/spec/$(ISE_PLATFORM)/lib/eiffel_curl.o">
+
+		<external_object location="$(ISE_LIBRARY)\library\cURL\spec\$(ISE_C_COMPILER)\$(ISE_PLATFORM)\lib\eiffel_curl.lib">
 			<condition>
-				<platform excluded_value="windows"/>
+				<platform value="windows"/>
 				<concurrency value="none"/>
+				<dotnet value="false"/>
 			</condition>
 		</external_object>
-		<external_object location="$(ISE_LIBRARY)/library/cURL/spec/$(ISE_PLATFORM)/lib/MTeiffel_curl.o">
+
+		
+		<external_object location="$(ISE_LIBRARY)\library\cURL\spec\$(ISE_C_COMPILER)\$(ISE_PLATFORM)\lib\eiffel_curl_static.lib">
 			<condition>
-				<platform excluded_value="windows"/>
-				<concurrency excluded_value="none"/>
+				<custom name="static_curl_enabled" value="true"/>
+				<platform value="windows"/>
+				<concurrency value="none"/>
+				<dotnet value="false"/>
 			</condition>
 		</external_object>
-		<external_object location="$(ISE_LIBRARY)\library\cURL\spec\$(ISE_C_COMPILER)\$(ISE_PLATFORM)\lib\eiffel_curl.lib">
+		<external_object location="$(ISE_LIBRARY)\library\cURL\spec\$(ISE_C_COMPILER)\$(ISE_PLATFORM)\lib\MTeiffel_curl_static.lib">
 			<condition>
+				<custom name="static_curl_enabled" value="true"/>
 				<platform value="windows"/>
-				<concurrency value="none"/>
+				<concurrency excluded_value="none"/>
 				<dotnet value="false"/>
 			</condition>
 		</external_object>
+
 		<external_object location="$(ISE_LIBRARY)\library\cURL\spec\$(ISE_C_COMPILER)\$(ISE_PLATFORM)\lib\mteiffel_curl.lib">
 			<condition>
+				<custom name="static_curl_enabled" excluded_value="true"/>
 				<platform value="windows"/>
 				<concurrency excluded_value="none"/>
 				<dotnet value="false"/>
 			</condition>
 		</external_object>
 		<external_object location="$(ISE_LIBRARY)\library\cURL\spec\$(ISE_C_COMPILER)\$(ISE_PLATFORM)\lib\ileiffel_curl.lib">
 			<condition>
+				<custom name="static_curl_enabled" excluded_value="true"/>
```

Comment:
I would suggest to put in the .ecf description (that we usually don't use enough) explanation about `static_curl_enabled` and also precise that by default `dynamic` behavior is used.

Reply:
Something like

Set `static_curl_enabled` to `true` in your app if you want to use cURL in static mode, by default `dynamic` mode is used.